packages/inventory/src/components/detail/InventoryDetail.js

```diff
@@ -12,6 +12,11 @@ import { addNotification } from '@redhat-cloud-services/frontend-components-noti
 import ApplicationDetails from './ApplicationDetails';
 import './InventoryDetail.scss';
 
+const fallbackInventoryId = () => {
+    const pathName = location.pathname.split('/');
+    return pathName.pop() || pathName.pop(); // if last partial is empty use second to last
```

Comment:
What about 

```
pathName[pathName.length - 1] || pathName[pathName.length - 2]
```

Reply:
That's even more confusing :(

Reply:
@karelhala why don't we just strip replace `/\/$/` with `''` from the pathname?

Reply:
I try to avoid replacing, as it can lead to confusion. But This can be simplified to oneliner.

Reply:
You can extract it to seperate functions `getlast(pathname) || getSecondLast(pathname)`

Reply:
Or make a function `getLastDefined(array)` and make it more generic.